### ISSUE TITLE
Can't configure Redshift with terminal

### ISSUE THREAD

**Issue by lambisboy** [jonls/redshift#473]
![image](https://user-images.githubusercontent.com/29781840/27700684-b13ecf26-5cd4-11e7-846a-3786f23c7020.png)
When I use any command on terminal to change the redshift's temp, it gets stuck in using "randr" method forever.

**Comment by EvertonMelo** [comment]
Hi, 

I suggest you change the settings of the file:

`~/.config/redshift.conf`

Remember the adjust is limeted:

> Redshift assumes that your screen will produce light at a color of 6500K when no color correction is applied by the program. Thus, 6500K is the neutral temperature. Setting the color temperature to a value higher than this results in more blue light, and setting a lower value will result in more red light.

Font: http://jonls.dk/redshift/ ( not is Shorted, this is official site ) 

Sussa?

**Comment by jonls** [comment]
@lambisboy From the screenshot it seems like you are trying to use oneshot mode (-o) but the command line should have been `redshift -o 3200`. This may explain why it's not working as expected?

**Comment by Wash-up** [comment]
I'm having the same problem. Every command that changes Redshift settings get stuck either on "Using method randr" or "Using provider geoclue" and the screen keeps flickering between reddish and the default color.

Also, I tried creating the file `~/.config/redshift.conf`and commanding Redshift to load the settings on that file (via terminal) but it get stuck the same way, only this same there wasn't any message.

**Comment by jonls** [comment]
@Wash-up This sounds like a separate problem from what is reported in this issue. Run with "-v" to get more information on what Redshift is doing (it is most likely not stuck anywhere it just doesn't provide any more output by default). Also, per the FAQ:

> Why does the redness effect continuously flicker?
>
> You may have multiple instances of Redshift running simultaneously. Make sure that only one instance is running for the display where you are seeing the flicker.

Make sure you have only one instance of Redshift running. Also, be aware that other programs may be running that affect the gamma ramps that Redshift cannot prevent. The issue you see is not likely to be fixed by upgrading to 1.11.

**Comment by Wash-up** [comment]
@jonls You're right, running with "-v" shows that Redshift is working, I thought that it was stucked because I was forced to close terminal to regain control (btw, is there a workaround on this?).

Still, the "-t" command is not working for me. When I run it, Redshift stops on "Using method randr" and after I reopen the terminal and run "-v" the color temperatures don't change.

Maybe it's better if I open another issue on this?

About the flickering, I reinstalled Redshift and it didn't happened anymore.
